agent/apps/files/app_code/views.py

```
@@ -0,0 +1,87 @@
+from .serializers import (
+    StoredFileSerializer,
+    StoredFileCreateSerializer,
+)
+
+
+class StoredFileViewSet(ModelViewSet):
+    """
+    ViewSet مدیریت فایل‌ها (آپلود/لیست/حذف)
+    """
+
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)